### ISSUE TITLE
On Moor 4.4.0, analyzer error

### ISSUE THREAD

**Issue by SteveAlexander** [simolus3/moor#1320]
On switching from moor 4.3.2 to 4.4.0, I get an error shortly after restarting the analyzer in VS Code. This appears in a small .pop-up window:

```
Socket error: type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'Map<String, Object>' in type cast

Source: Dart (Extension)
```

The problem goes away if I switch back to 4.3.2 and restart the analyzer

**Comment by simolus3** [comment]
Wait, so the analyzer crashes? Are you using the moor analysis plugin?

**Comment by SteveAlexander** [comment]
yes.

```
analyzer:
  plugins:
    - moor
```

**Comment by simolus3** [comment]
Thanks for the report! I've fixed some of those problems in 5a595d251949c7d209edeede0d490a45240f874e.  I'll release those fixes this weekend.

**Comment by simolus3** [comment]
This should be fixed in moor version 4.4.1, which I've just published.
